index.bs

```diff
@@ -2627,6 +2656,14 @@ optionally evidence of [=user consent=] to a specific transaction.
 
         The default value is {{AttestationConveyancePreference/none}}.
 
+    :   <dfn>attestationFormats</dfn>
+    ::  The [=[RP]=] MAY use this OPTIONAL member to specify a preference regarding the [=attestation=] statement format used by the [=authenticator=].
+        Values SHOULD be taken from the IANA "WebAuthn Attestation Statement Format Identifiers" registry [[!IANA-WebAuthn-Registries]] established by [[!RFC8809]].
+        Values are ordered from most preferable to least preferable.
+        This parameter is advisory and the [=authenticator=] MAY use an attestation statement not enumerated in this parameter.
+
+        The default value is the empty list, which indicates no preference.
+
```

Comment:
During credential creation, does this attestationFormats parameter refer to the preference of the attestation of the primary credential, the DPK extension, or both? That seems ambiguous, and it is not clear what the RP uses this parameter for.